ignite/contrib/handlers/custom_events.py

```diff
@@ -51,8 +52,8 @@ def on_every_10_epochs(engine):
 
     def __init__(self, n_iterations=None, n_epochs=None):
         
-        raise DeprecationWarning("CustomPeriodicEvent is deprecated since v0.4.0 and"
-                                 "removing since v0.5.0, use filtered events instead")
+        warnings.warn("CustomPeriodicEvent is deprecated since v0.4.0 and removing since v0.5.0"
```

Comment:
Nit: `"CustomPeriodicEvent is deprecated since 0.4.0 and will be removed in 0.5.0. Use filtered events instead."`